src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertIntegrationTest.java

```diff
@@ -1309,9 +1309,9 @@ private void _verifyInsert(String docId, JsonNode doc) {
 
   @Nested
   @Order(4)
-  class InsertOneFails {
+  class InsertOneWithPathConflict {
```

Comment:
Maybe add a comment explaining why there is no real conflict for seemingly overlapping paths (or change name to "WithOverlappingPaths" to remove "Conflict" part)